pkg/executor/sortexec/parallel_sort_spill_helper.go

```diff
@@ -52,7 +52,6 @@ func newParallelSortSpillHelper(sortExec *SortExec, fieldTypes []*types.FieldTyp
 		errOutputChan: errOutputChan,
 		finishCh:      finishCh,
 		fieldTypes:    fieldTypes,
-		tmpSpillChunk: chunk.NewChunkFromPoolWithCapacity(fieldTypes, spillChunkSize),
```

Comment:
@hawkingrei ptal. related to https://github.com/pingcap/tidb/pull/53859

Reply:
LGTM

Reply:
Can we reset the requiredRows for chunk got from NewChunkFromPoolWithCapacity? @xzhangxian1008

Reply:
> Can we reset the requiredRows for chunk got from NewChunkFromPoolWithCapacity? @xzhangxian1008

yes, I have reset it in this pr.